pkg/kudoctl/cmd/plan/plan_status.go

```diff
@@ -52,6 +53,11 @@ func runStatus(cmd *cobra.Command, args []string, options *statusOptions) error
 		return fmt.Errorf("flag Error: Please set instance flag, e.g. \"--instance=<instanceName>\"")
 	}
 
+	// If the $KUBECONFIG environment variable is set, use that
+	if len(os.Getenv("KUBECONFIG")) > 0 {
```

Comment:
Would it make sense to have a `validate()` similar like in https://github.com/kudobuilder/kudo/pull/479/files#diff-47795ca62796b678e9c01915094311ddR36 to use similar patterns, maybe even moving the `validate` into its own util function?

Reply:
I wanted to at least get the behaviour correct and consistent first of all across these commands, and then refactor - possibly into the check package.  Sound sensible?

Reply:
make sense! :) lgtm